src/components/WishlistPage/WishlistPage.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import propTypes from 'prop-types';
+
+import WishlistItem from './WishlistItem';
+
+import './WishlistPage.scss';
+
+
+const CN = 'wishlist';
+
+class WishlistPage extends Component {
```

Comment:
when you don't use state or lifecycle methods please make it a function component